README.md

```diff
@@ -39,50 +39,51 @@ to deliver the FAT32 Filesystem/SDCard support. This document explains how to do
+- How the software components work together to deliver the storage functionality.
 - How to work with the sd-driver and mbed OS to build the examples. The example code can easily
   be copied into your new application code.
 - How to work with the CI Test Shield, which adds an SDCard slot to those targets that do not have already have one.
 - How to run the POSIX File API mbed Greentea test cases, which provide further example code of how to use
   the POSIX File API.
- 
+
 Section 1 provides an Executive Summary, describing the purpose of the sd-driver, the supporting
 software, examples, test cases and documentation.
 
-Section 2 provides an an overview of the material covered including descriptions of the major sections. 
+Section 2 provides an an overview of the material covered including descriptions of the major sections.
 
-Section 3 provides an overview of the mbed OS filesystem software components, 
-including the inter-relationships between the application, POSIX file API, the standard c-library, 
-the mbed OS filesystem and the SDCard driver (sd-driver). 
+Section 3 provides an overview of the mbed OS filesystem software components,
+including the inter-relationships between the application, POSIX file API, the standard c-library,
+the mbed OS filesystem and the SDCard driver (sd-driver).
 
-Section 4 describes how to build and run an example application for reading 
+Section 4 describes how to build and run an example application for reading
 and writing data to an SDCard using the POSIX File API. The example begins by describing
 the procedure for building and testing on the K64F target. The final sub-sections
-describe how to use the test shield to add an SDCard slot to any mbed target, 
+describe how to use the test shield to add an SDCard slot to any mbed target,
 and hence enable the persistent storage of data on any supported target.
 
-Section 5 describes an example application which uses the raw 
+Section 5 describes an example application which uses the raw
 BlockDevice API to read and write data to the SDCard.
 
 Section 6 describes how to build and run the SDCard POSIX File API mbed Greentea test cases.
-There are a number of functional test cases demonstrating how to use the 
-mbed OS POSIX File API. 
+There are a number of functional test cases demonstrating how to use the
+mbed OS POSIX File API.
 
-Section 7 describes the POSIX File API and provides links to useful API documentation web pages. 
+Section 7 describes the POSIX File API and provides links to useful API documentation web pages.
 
 
 ### Known mbed-os and sd-driver Compatible Versions
 
-The following versions of the mbed-os and sd-driver repositories are known to work together: 
+The following versions of the mbed-os and sd-driver repositories are known to work together:
 
-- {mbed-os, sd-driver} = {mbed-os-5.4.0-rc2, sd-driver-0.0.1-mbed-os-5.4.0-rc2}. 
+- {mbed-os, sd-driver} = {mbed-os-5.4.0-rc2, sd-driver-0.0.1-mbed-os-5.4.0-rc2}.
```

Comment:
Versions is updated few lines below in continuity
'{mbed-os, sd-driver} = {mbed-os-5.5.1, sd-driver-0.1.0-mbed-os-5.5.1}.'

Above lines are for old versions as in Readme, change is in formatting and extra spacing characters at the end of line. Sorry for confusion.